pkg/operator/configobservation/network/observe_network.go

```diff
@@ -80,11 +80,10 @@ func ObserveServicesSubnet(genericListers configobserver.Listers, recorder event
 	listers := genericListers.(configobservation.Listers)
 
 	out := map[string]interface{}{}
-	servicesSubnetConfigPath := []string{"servicesSubnet"}
-	bindAddressConfigPath := []string{"servingInfo", "bindAddress"}
-	bindNetworkConfigPath := []string{"servingInfo", "bindNetwork"}
+	servicesSubnetConfigPath := []string{"apiServerArguments", "service-cluster-ip-range"}
+	bindAddressConfigPath := []string{"apiServerArguments", "bind-address"}
```

Comment:
on the downgrade, the old operator will just replace it with `servingInfo.bindAddress`,  right?

Reply:
> on the downgrade, the old operator will just replace it with `servingInfo.bindAddress`, right?

yeah, no impact to the old operator.